setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = odc-geo
+name = odc_geo
```

Comment:
Why the name change?

Reply:
warning from `pip` in python the two are equivalent anyway.

Reply:
From which command? I just tried `pip install -e .` and `python -m build` and neither gave me a warning. IMO it feels better to have the name match the github slug?

Reply:
@gadomski warning was coming from `options.package_data` not from package name, I have reverted package name change and fixed package_data section.